gtfs/spec/en/reference.md

```diff
@@ -612,6 +623,69 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### location_groups.txt
+
+File: **Optional**
+
+Primary key (`location_group_id`, `location_id`)
+
+Defines location groups that indicate groups of stops where a rider may request pickup or drop off.
+
+| Field Name | Type | Required | Description |
+| ---------- | ---- | ------------ | ----------- |
+| `location_group_id` | ID | **Required** | Identifies a location group. A location group is a group of stops that together indicate locations where a rider may request pickup or drop off.<br><br> By default, every `stop_id` belongs to a `location_group_id` of the same value. Therefore, it is forbidden to define a `location_group_id` with the same value as a `stop_id`.<br><br>Multiple entries in `location_groups.txt` can have the same `location_group_id`. | 
+| `stop_id` | Foreign ID referencing `stops.stop_id` | **Required** | Identifies a stop belonging to the location group. |
+| `location_group_name` | Text | Optional | Name of the location group. Must be defined either once, or exhaustively for a single `location_group_id`. |
+
+### locations.geojson
+
+File: **Optional**
+
+Defines GeoJSON locations that indicate groups of lat/lon coordinates defining zones where riders can request either pickup or drop off.
+- This file uses a subset of the GeoJSON format, described in [RFC 7946](https://tools.ietf.org/html/rfc7946).
+- The `locations.geojson` file must contain a `FeatureCollection`.
+- A `FeatureCollection` defines various stop locations where riders may request pickup or drop off.
+- Every GeoJSON `Feature` must have an `id`. The `id` belongs to the same namespace as `stop_id` in `stops.txt` and `location_group_id` in `location_groups.txt`.
+- Every GeoJSON `Feature` should have objects and associated keys according to the table below:
+
+|  Field Name | Type | Presence | Description |
+|  ------ | ------ | ------ | ------ |
+| -&nbsp;`type` | String | **Required** | `"FeatureCollection"` of locations. |
+| -&nbsp;`features` | Array | **Required** | Collection of `"Feature"` objects describing the locations. |
+| &nbsp;&nbsp;&nbsp;&nbsp;\-&nbsp;`type` | String | **Required** | `"Feature"` |
+| &nbsp;&nbsp;&nbsp;&nbsp;\-&nbsp;`id` | String | **Required** | Location ID belonging to the same namespace as `stops.stop_id`. It is forbidden to define an `id` from `locations.geojson` with the same value as a `stops.stop_id` or a `location_groups.location_group_id`.|
```

Comment:
I recommend removing "It is forbidden to define an `id` from `locations.geojson` with the same value as a `stops.stop_id` or a `location_groups.location_group_id`."

We no longer need to establish this restriction since they do not share the stop_times.stop_id field anymore.

Reply:
Also, see my comment on line 648 regarding "namespace" language.

Reply:
Commit pushed - https://github.com/google/transit/pull/388/commits/6314252ed5c422e6d229654dfac282a3c5ca0ce3
Thanks!